generators/kubernetes/templates/_service.yml

```diff
@@ -20,11 +20,19 @@ apiVersion: v1
 kind: Service
```

Comment:
I think it is OK to have it in the same file. In a real time case, say in production if something needs to be changed, then we most likely tend to apply the change by patching it through cli or the console. I mean you wouldn't really delete a service object and recreate it in production right? For eg: if we want to change the selector or service type we more likely prefer to use kubectl cli. And moreover service object is a logical wrapper to reach to the deployed service instances. To me logically grouping these objects makes sense as it all together represent a single app entity.